diff --git a/node_modules/react-native-raw-bottom-sheet/index.d.ts b/node_modules/react-native-raw-bottom-sheet/index.d.ts
index 31a5259..0e078b9 100644
--- a/node_modules/react-native-raw-bottom-sheet/index.d.ts
+++ b/node_modules/react-native-raw-bottom-sheet/index.d.ts
@@ -1,4 +1,4 @@
-import { Component } from "react";
+import { PropsWithChildren, Component } from "react";
 import { StyleProp, ViewStyle } from "react-native";
 
 declare module "react-native-raw-bottom-sheet" {
@@ -22,7 +22,7 @@ declare module "react-native-raw-bottom-sheet" {
     keyboardAvoidingViewEnabled?: boolean;
   };
 
-  export default class RBSheet extends Component<RBSheetProps> {
+  export default class RBSheet extends Component<PropsWithChildren<RBSheetProps>> {
     open(): void;
     close(): void;
   }
